app/src/main/java/org/wikipedia/notifications/NotificationActivity.kt

```diff
@@ -266,24 +273,32 @@ class NotificationActivity : BaseActivity() {
             if (!currentSearchQuery.isNullOrEmpty() && n.contents != null && !n.contents.header.contains(currentSearchQuery!!)) {
                 continue
             }
-            notificationContainerList.add(NotificationListItemContainer(n))
+            val filterList = mutableListOf<String>()
+            filterList.addAll(StringUtil.csvToList(Prefs.notificationsFilterLanguageCodes.orEmpty()).filter { NotificationCategory.isFiltersGroup(it) })
+            if (filterList.contains(n.category) || Prefs.notificationsFilterLanguageCodes == null) notificationContainerList.add(NotificationListItemContainer(n))
```

Comment:
Can we use `.isNullOrEmpty()` on the `Prefs.notificationsFilterLanguageCodes`?

Reply:
Same as above